docs/src/howto-guides/002_installation.md

```diff
@@ -103,19 +103,6 @@ Eg:
 dpkg -i tedge_mapper_0.5.0_armhf.deb
 ```
 
-### Add your user to `tedge-users` group
-
-During the installation process, a `tedge-users` group is automatically created,
-in order to ease the administration of who can use the `sudo tedge` command on the device.
-Indeed, the `tedge` command needs to be run using `sudo`.
-So, unless all the users are granted sudo privileges, you have to add a user to the `tedge-users` group for that user to be able to use `tedge`.
-
-Run this command to add a user to the group.
-
-```shell
-sudo adduser <user> tedge-users
-```
```

Comment:
This documentation can be removed only if we choose the option where no user is added to the `tedge` group to gain `sudo` access to `tedge` (the 2nd alternative of my comment above on `configuration/debian/tedge/postinst`)

Reply:
Yes, that was the idea. I wanted just to have `tedge` user and no other user can be added to the `tedge` user group.